pkg/lib/serverTime.js

```diff
@@ -258,42 +267,29 @@ export function ServerTime() {
         return status;
     };
 
-    const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
-                                                { superuser: "try" });
+    function get_timesync_backend() {
+        return python.spawn(get_timesync_backend_py, [], { superuser: "try", err: "message" })
+                .then(data => {
+                    const unit = data.trim();
+                    if (unit == "systemd-timesyncd.service")
+                        return "timesyncd";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18298-20230308-104315-31f0d449-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)